pkg/packet/bgp/bgp.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 type MarshallingOption struct {
-	AddPath map[RouteFamily]BGPAddPathMode
+	AddPath    map[RouteFamily]BGPAddPathMode
+	Attributes *BGPUpdateAttributes
```

Comment:
using `map[BGPAttrType]bool` is simpler?

Reply:
Thank you for the suggestion, I will change it to use map directly.